evap/student/views.py

```diff
@@ -105,12 +105,51 @@ def sorter(evaluation):
         )
 
     unfinished_evaluations.sort(key=sorter)
+    info = Evaluation.annotate_with_participant_and_voter_counts(
+        Semester.active_semester()
+        .evaluations.filter(is_single_result=False)
+        .exclude(state__lt=Evaluation.State.APPROVED)
+        .exclude(is_rewarded=False)
+        .exclude(id__in=settings.EXCLUDED_EVALUATION_IDS)
+        .exclude(course__type__id__in=settings.EXCLUDED_COURSE_TYPE_IDS)
+        .exclude(course__is_private=True)
+    ).aggregate(
+        total_votes=Sum("num_voters"),
+        participants=Sum("num_participants"),
```

Comment:
"total_" vs no prefix is inconsistent
```suggestion
        total_votes=Sum("num_voters"),
        total_participants=Sum("num_participants"),
```